zip-1012.rst

```diff
@@ -3,17 +3,21 @@
   ZIP: 1012
```

Comment:
Could we publish earlier and move to ZIP 1014 @daira @gtank ? @zookozcash and I think for polling it would be nicer for folks to see it on https://zips.z.cash/ . I'd be happy to accept all these changes and get it ready to make it an easier PR.